app/models/request.js

```diff
@@ -17,6 +17,7 @@ export default Model.extend({
   headCommit: attr(),
   baseCommit: attr(),
   branchName: attr(),
+  pullRequestMergeable: attr(),
```

Comment:
Am I right that this field is `string`? If so, please specify the type for `attr`:
```js
pullRequestMergeable: attr('string'),
```
This helps to ensure the field type and derialize/extract data properly. More info about it here: https://guides.emberjs.com/release/models/defining-models/#toc_transforms